src/main/java/via/dk/cueandbrew/model/Reservation.java

```diff
@@ -0,0 +1,82 @@
+package via.dk.cueandbrew.model;
+
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Reservation {
+    private String notes;
+    private String clientFirstName;
+    private String clientLastName;
+    private String clientPhoneNumber;
+    private Timestamp creationDatetime;
+    private List<Booking> booking;
+    private List<Order> order;
+
+    public Reservation() {
```

Comment:
Idk if this is relevant here, but maybe we could try to implement the Builder Pattern for this